--- "recovered/End of Run Summary.gd"	2026-05-09 11:03:48.329854360 +1000
+++ "recovered/End of Run Summary.gd"	2026-05-09 11:04:20.979855270 +1000
@@ -175,6 +175,12 @@
         %"Moeny UI".animate()
         %"TaDa Audio".play()
     )
+    if Global.AUTO_ADVANCE_END:
+        anim_tween.tween_interval(Global.DELAY_SPEED)
+        anim_tween.tween_callback(func():
+            if is_shown and Global.game_state == Util.GAME_STATES.END_OF_SESSION:
+                _on_upgrades_pressed()
+        )
 
 
 func kill_tweens():
